StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/STPAPIClient+PaymentSheet.swift

```diff
@@ -91,6 +91,7 @@ extension STPAPIClient {
 extension STPAPIClient {
     typealias STPPaymentIntentWithPreferencesCompletionBlock = ((Result<STPPaymentIntent, Error>) -> Void)
     typealias STPSetupIntentWithPreferencesCompletionBlock = ((Result<STPSetupIntent, Error>) -> Void)
+    typealias STPElementsSessionWithPreferencesCompletionBlock = ((Result<STPElementsSession, Error>) -> Void)
```

Comment:
I think we should remove "withPreferences", so `STPElementsSessionCompletionBlock`.

It doesn't make sense in the above completion blocks either, but we can leave that alone for now.